.lgtm.yml

```diff
@@ -1,5 +1,4 @@
 path_classifiers:
   test:
     - test
-    - scripts
-  library:
\ No newline at end of file
+    - scripts
```

Comment:
there is a newline missing at the end of this file.

(at some point it would probably be wise to create a `pre-commit` configuration that automatically fixes such mistakes. See https://github.com/cda-tum/qcec/blob/main/.pre-commit-config.yaml and the corresponding PR that added the functionality https://github.com/cda-tum/qcec/pull/100. Note that the corresponding PR added much more functionality for ensuring the quality of the repository code. Technically you could create a separate issue for that)

Reply:
Done.

Issue is created for the pre-commit config.